content/en/docs/setup/platform-setup/prerequisites/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Platform Prerequisites
+description: Prerequisites for platform setup for istio.
+weight: 1
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/prerequisites/
+    - /docs/setup/kubernetes/platform-setup/prerequisites/
+keywords: [platform-setup,prerequisites]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+
+## Kernel Module Requirements on Cluster Nodes
+
+The following kernel modules must be loaded on the cluster node running application pods with a Service Mesh proxy sidecar container.
+The modules are needed specifically by the `istio-init` container which sets up iptables rules in the pod to redirect any incoming or
```

Comment:
Or istio-cni deameon set. 

Also add that the modules are needed for iptables interception - Istio can also work in 'whitebox mode' where it doesn't intercept and doesn't need any special kernel module ( and in future may also use ebpf or other mechanisms ). Maybe a section describing the interception modes - and a table for each ?

I believe 'raw' is only used for DNS interception.